src/deepsparse/transformers/server/schemas.py

```diff
@@ -166,4 +167,5 @@ class TextClassificationResponse(TaskResponseModel):
     "token-classification": Union[
         List[TokenClassificationResponse], List[List[TokenClassificationResponse]]
     ],
+    None: None,
```

Comment:
let's access this dict by `{}.get` instead of doing this

Reply:
Removed!